compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CompilerPluginRunner.java

```diff
@@ -233,6 +234,12 @@ public void visit(BLangSimpleVariable varNode) {
         notifyProcessors(attachmentList, (processor, list) -> processor.process(varNode, list));
     }
 
+    @Override
+    public void visit(BLangTupleVariable tupleVariableNode) {
+        List<BLangAnnotationAttachment> attachmentList = tupleVariableNode.getAnnotationAttachments();
+        notifyProcessors(attachmentList, (processor, list) -> processor.process(tupleVariableNode, list));
```

Comment:
Was trying to test the test compiler plugin and came across the following issue.

Adding the following in tests/ and doing a `ballerina build` results in a bad, sad error.
```ballerina
const annotation annot on source var;

@annot
[int, int] [x, y] = [1, 2];
```

```log
[2020-11-10 23:54:22,205] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CompilerPluginRunner.notifyProcessors(CompilerPluginRunner.java:313)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CompilerPluginRunner.visit(CompilerPluginRunner.java:240)
	at org.wso2.ballerinalang.compiler.tree.BLangTupleVariable.accept(BLangTupleVariable.java:56)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CompilerPluginRunner.visit(CompilerPluginRunner.java:152)
	at org.wso2.ballerinalang.compiler.tree.BLangPackage.accept(BLangPackage.java:163)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CompilerPluginRunner.runPlugins(CompilerPluginRunner.java:135)
	at org.wso2.ballerinalang.compiler.CompilerDriver.annotationProcess(CompilerDriver.java:367)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compile(CompilerDriver.java:303)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compilePackageSymbol(CompilerDriver.java:260)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compilePackage(CompilerDriver.java:143)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackages(Compiler.java:176)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackage(Compiler.java:221)
	at org.wso2.ballerinalang.compiler.Compiler.compile(Compiler.java:101)
	at org.wso2.ballerinalang.compiler.Compiler.build(Compiler.java:109)
	at org.ballerinalang.packerina.task.CompileTask.execute(CompileTask.java:62)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.BuildCommand.execute(BuildCommand.java:436)
	at java.base/java.util.Optional.ifPresent(Optional.java:183)
	at org.ballerinalang.tool.Main.main(Main.java:57)
```

Reply:
fixed